API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/IResourceUtilizationPublisher.cs

```diff
@@ -0,0 +1,11 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public interface IResourceUtilizationPublisher
+{
+    ValueTask PublishAsync(Utilization utilization, CancellationToken cancellationToken);
```

Comment:
Why ValueTask over Task?

Reply:
The implementation of these calls mostly complete synchronously.